lib/active_model/serializer.rb

```diff
@@ -55,7 +55,7 @@ def self.digest_caller_file(caller_line)
 
       serializer.class_attribute :_cache         # @api private : the cache object
       serializer.class_attribute :_fragmented    # @api private : @see ::fragmented
-      serializer.class_attribute :_cache_key     # @api private : when present, is first item in cache_key
+      serializer.class_attribute :_cache_key     # @api private : when present, is first item in cache_key.  Ignored if the serialized object defines #cache_key.
```

Comment:
in the diff, to change `Ignored if the serialized object defines #cache_key.` to `Ignored if the serializable object defines #cache_key.`